### ISSUE TITLE
Improve Profile design

### ISSUE THREAD

**Issue by cjyabraham** [cncf/wordpress-site#56]
- [x] Make [gear in top-right](https://www.cncf.io/speaker/prydonius/) clearer in what it does.  Maybe use the "Edit profile" buttons from the [directory page](https://www.cncf.io/speakers/)?
- [ ] Add badges for the certifications owned by the speaker

Note: the profile page can be customized using [this method](https://docs.ultimatemember.com/article/120-adding-your-custom-profile-templates)

**Comment by KimMcMahon** [comment]
@deigolocco @pablodelarocha 
Ideally, I would like the badges for each speaker on their individual profile page. Example - Ariel's badges would appear on his profile [page](https://www.cncf.io/speaker/arielmunafi/).

Before any work is done, I would like to know if / how this would be done, can the existing plug-in be used, if not using the existing plug-in what is the recommendation. Please provide an estimate before proceeding with design.

**Comment by deigolocco** [comment]
@KimMcMahon the plugin allow some customizations on it templates. Is not a complex task, can be done in a few ours, and this will not affect the plugin's functionalities. Soon @pablodelarocha will send a design proposal.

**Comment by pablodelarocha** [comment]
![speaker_page](https://user-images.githubusercontent.com/54405572/66337725-14430680-e916-11e9-8a7a-2fabfb3cb435.jpg)
Here's a suggestion for edit profile icon at the top, instead of the gear and also of how to show the badges. Please let us know if this is ok to proceed.

**Comment by rodmeurer** [comment]
@KimMcMahon can we proceed with this?

**Comment by cjyabraham** [comment]
My 2c: I prefer the gear icon.  I think it's more obviously the place I'd go if I was looking to edit my profile.

And another design option to explore: just having the certification badges in place of the word "Ambassador" in your mockup, instead of where you have them.

**Comment by KimMcMahon** [comment]
@rodmeurer Sorry - I forgot to comment!

I agree with @cjyabraham - please keep the Gear icon.

I also agree that we put the badges under their name (as you have) and remove Certifications on the left side. Keep the Travel Range that is on the left side.

Can you do another mockup for review?

Thanks!

**Comment by pablodelarocha** [comment]
Yeah, sure. Let us know if we may proceed with the below design:
![speaker_page](https://user-images.githubusercontent.com/54405572/66661340-38a42900-ec1d-11e9-8bd5-32c54ffb8441.jpg)

**Comment by KimMcMahon** [comment]
Looks great @pablodelarocha! Please proceed with the development.

Thanks!
Kim

**Comment by KimMcMahon** [comment]
@pablodelarocha Is this ready to test so we can deploy?

**Comment by deigolocco** [comment]
yes @KimMcMahon https://fz-56-cncf.pantheonsite.io/speaker/janakiramm/

**Comment by KimMcMahon** [comment]
@deigolocco looks good. This profile should have an icon to symbolize Sig-Storage.
https://fz-56-cncf.pantheonsite.io/speaker/chira001/

For SIG Security, use this image. Can you suggest an icon for SIG Storage and SIG App Delivery?
![sig-security-icon-color](https://user-images.githubusercontent.com/29307455/67207309-86185700-f3d0-11e9-9897-6d8800bad865.png)

**Comment by dankohn** [comment]
@KimMcMahon Please talk to @amye as they'll be coming up with their own logos soon. 

If you need something in the meantime, you can use these emoji:

App Delivery: 📨🚚
Storage: 🗄

**Comment by KimMcMahon** [comment]
@deigolocco Please use the emoji's Dan shared

@dankohn Amye and I talked - she will let me know when they have a logo.

**Comment by cjyabraham** [comment]
@KimMcMahon @deigolocco , looking at this demo, I don't like how you have to hover over the logos in order to figure out what they are.  Would you be open to making them larger so that the text can be read in the logos and, perhaps, placing them to the right of the photo?

**Comment by pablodelarocha** [comment]
@cjyabraham heres a couple of options. I guess the badges can't be smaller than this to still be readable. If we make them larger, then it might be a problem in profiles that has all certifications.
Also, i'm suggesting that we limit the # of characters on the bio and title, and add a "Read more" link to expand it. 
Also the plane and train icons, i'm suggesting they still appear below the title and small, less prominent than the badges.
![02](https://user-images.githubusercontent.com/54405572/67297144-e0441580-f4bf-11e9-95e6-6aa9bfc44df7.jpg)
![01](https://user-images.githubusercontent.com/54405572/67297146-e0441580-f4bf-11e9-800d-2d93af025452.jpg)

**Comment by cjyabraham** [comment]
@pablodelarocha hmm, can you try one other option iterating from your first solution above:
- put the bio back down to where it is in [our current profiles](https://www.cncf.io/speaker/seungkyua/)
- put the title field text in the right-column above the certifications

In this configuration I don't think we'll need read more links.

**Comment by pablodelarocha** [comment]
@cjyabraham  sure, here's a couple of new suggestions as you requested. Let us know if any of these works for you.

![03](https://user-images.githubusercontent.com/54405572/67311284-a716a000-f4d5-11e9-857a-8a88dfab39ce.jpg)
![04](https://user-images.githubusercontent.com/54405572/67311285-a7af3680-f4d5-11e9-8046-8ecc4eacd309.jpg)

**Comment by cjyabraham** [comment]
I like the second one, except I don't like where the airplane icon is.  I wonder if we need that icon at all since we have that info below in the Travel Range field and I don't think that info really needs to be so prominent.  So my preference would be the second option without the airplane icon.

@KimMcMahon @dankohn ?

**Comment by dankohn** [comment]
LGTM. Let's put the plane in front of the word International at the bottom left.

**Comment by pablodelarocha** [comment]
Ok @dankohn and @cjyabraham we'll proceed with the below design then. Thanks!
![profile](https://user-images.githubusercontent.com/54405572/67391583-60788280-f575-11e9-8bc8-2e451f7d939d.jpg)

**Comment by rodmeurer** [comment]
Done @cjyabraham 

https://fz-56-cncf.pantheonsite.io/speaker/jgarr/

**Comment by cjyabraham** [comment]
Nice.  A few comments:
1. Can you make the text used in the title to be the same font, size etc. as that used for the bio?
2. Speakers without a title have [some](https://fz-56-cncf.pantheonsite.io/speaker/prydonius/) [render](https://fz-56-cncf.pantheonsite.io/speaker/carlisia/) issues
3. When you hover on a badge, the mouse cursor changes making it seem like you can click on them, but you can't.  @dankohn do we want to link the badges to some information on them or a search for speakers with that badge?  Or just leave them unlinked?
4. If the speaker has [no certifications](https://fz-56-cncf.pantheonsite.io/speaker/aberoham/) then the "Certifications" banner should not show up.  If there is no title or certifications then the photo can be centered.

**Comment by dankohn** [comment]
Unlinked is fine.

**Comment by rodmeurer** [comment]
@cjyabraham fixed.

https://fz-56-cncf.pantheonsite.io/speaker/carlisia/
https://fz-56-cncf.pantheonsite.io/speaker/prydonius/
https://fz-56-cncf.pantheonsite.io/speaker/aberoham/
https://fz-56-cncf.pantheonsite.io/speaker/jgarr/

**Comment by cjyabraham** [comment]
Almost there.  Can you look at the alignment of the [photo and title here](https://fz-56-cncf.pantheonsite.io/speaker/liviucostea/)?  They look too far right and don't match the alignment of the Bio section.

**Comment by KimMcMahon** [comment]
@cjyabraham The photo and title you reference looks good to me. Can you please deploy.

As a FYI - I'm waiting on sending the second email to speakers until this new design is ready. Thank you.

**Comment by deigolocco** [comment]
@cjyabraham and @KimMcMahon we finished the adjustments yesterday

**Comment by cjyabraham** [comment]
@deigolocco alignment looks better, thx.

Can you use [the logos Dan mentioned](https://github.com/cncf/wordpress-site/issues/56#issuecomment-544533881) for SIG App Delivery, SIG Security and SIG Storage?  Will they work?  I'm seeing Sig Storage is missing from [here](https://fz-56-cncf.pantheonsite.io/speaker/chira001/) and [all three from here](https://fz-56-cncf.pantheonsite.io/speaker/randy.abernethy/).

@KimMcMahon we should get these things fixed before we deploy and publicize.

**Comment by rodmeurer** [comment]
@cjyabraham we used the logos as suggested, let us know if it is as expected. We couldn't get the SVG version for the "delivery" logo.

**Comment by cjyabraham** [comment]
Thanks @rodmeurer .  

@dankohn @KimMcMahon the logos we have for SIG App Delivery and SIG Storage [are not self-explanatory](https://fz-56-cncf.pantheonsite.io/speaker/randy.abernethy/).  Do you suggest we add text below them or wait for something recognizable form @amye ?

**Comment by KimMcMahon** [comment]
@cjyabraham @deigolocco 
At one point, did we have the words pop up with a hover over the icon? I thought I saw a conversation on that, but I don't recall the outcome.

Change "certifications" to "affiliations" on the individual speaker page.
<img width="806" alt="Screen Shot 2019-10-29 at 1 49 48 AM" src="https://user-images.githubusercontent.com/29307455/67747507-7b704a00-f9ee-11e9-95a1-c2ea8e708dfe.png">

**Comment by cjyabraham** [comment]
Yes, we had hover tooltips before when the other icons were very small.  We could have them again for these ones that don't have text but not needed now for the others?

**Comment by deigolocco** [comment]
@KimMcMahon please confirm, if necessary we can add the tooltip back

**Comment by KimMcMahon** [comment]
@deigolocco We are going to have custom icons in the next couple of weeks. Let's not worry about tooltips for now. If there is confusion, we can add them back.

**Comment by deigolocco** [comment]
ok, thanks @KimMcMahon 

@cjyabraham I believe we are good to deploy

**Comment by cjyabraham** [comment]
@KimMcMahon I think there will immediately be confusion if we deploy as is.  I recommend we use tooltips, have text under the ambiguous icons, or even just have text and no icons for now.  Otherwise, if we deploy this and then announce it, people will immediately start filling in their certifications and will be confused by mail-inbox icons and filing-cabinet icons, IMHO.  Some will complain about it, other will just figure it doesn't work and will never come back.  For this kind of thing we may only have one shot at getting people to trust and participate so I think we need to give our best from the start. What do you think?

**Comment by cjyabraham** [comment]
@dankohn do you have thoughts on this ^ ?

**Comment by dankohn** [comment]
I would suggest that we remove the SIGs for now. They are different than being a CKA or ambassador. We could decide later to allow people to select SIGs and projects that they're associated with, but that probably fits better in a bio.

**Comment by cjyabraham** [comment]
Ok.  I will deploy.

**Comment by cjyabraham** [comment]
this is deployed.
